modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -79,7 +79,7 @@ You can install {product-title} version {product-version} on the following IBM h
 
 [NOTE]
 ====
-You can use dedicated or shared IFLs to assign sufficient compute resources. Resource sharing is one of the key strengths of IBM Z. However, you must adjust capacity correctly on each hypervisor layer and ensure sufficient resources for every {product-title} cluster.
+You can use dedicated or shared IFLs to assign sufficient compute resources. Resource sharing is one of the key strengths of {ibmzProductName}. However, you must adjust capacity correctly on each hypervisor layer and ensure sufficient resources for every {product-title} cluster.
```

Comment:
Should the first usage of IFL be spelled out? (I don't think this is the first usage, but I didn't find it spelled out in the other instances, either.)